src/components/ActionMenu/ActionMenu.tsx

```diff
@@ -59,10 +59,6 @@ export class ActionMenu extends React.PureComponent<ActionMenuProps, State> {
     const {activeMenuGroup} = this.state;
     const menuActions = [...actions, ...groups];
 
-    if (menuActions.length === 0) {
```

Comment:
This branch was never executing because we're [returning early](https://github.com/Shopify/polaris-react/blob/5c84213f58e8c44a07d23c214aad59a709c1c0b6/src/components/ActionMenu/ActionMenu.tsx#L35) in the render method